src/utils/aadGroup.ts

```diff
@@ -48,5 +50,25 @@ export const aadGroup = {
     }
 
     return groups[0];
+  },
+
+  /**
+   * Get a single group id by its display name.
+   * @param displayName Group display name.
+   * @throws Error when group was not found.
+   * @throws Error when multiple groups with the same name were found.
+   */
+  async getGroupIdByDisplayName(displayName: string): Promise<string> {
+    const groups = await odata.getAllItems<Group>(`${graphResource}/v1.0/groups?$filter=displayName eq '${formatting.encodeQueryParameter(displayName)}'?$select=id`);
```

Comment:
When executing this function, I'm getting an error because the URL is invalid.

```suggestion
    const groups = await odata.getAllItems<Group>(`${graphResource}/v1.0/groups?$filter=displayName eq '${formatting.encodeQueryParameter(displayName)}'&$select=id`);
```